lib/stdlib/src/erl_tar.erl

```diff
@@ -14,198 +14,252 @@
 %% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 %% See the License for the specific language governing permissions and
 %% limitations under the License.
-%% 
+%%
 %% %CopyrightEnd%
 %%
+%% This module implements extraction/creation of tar archives.
+%% It supports reading most common tar formats, namely V7, STAR,
+%% USTAR, GNU, BSD/libarchive, and PAX. It produces archives in USTAR
+%% format, unless it must use PAX headers, in which case it produces PAX
+%% format.
+%%
+%% The following references where used:
+%%   http://www.freebsd.org/cgi/man.cgi?query=tar&sektion=5
+%%   http://www.gnu.org/software/tar/manual/html_node/Standard.html
+%%   http://pubs.opengroup.org/onlinepubs/9699919799/utilities/pax.html
 -module(erl_tar).
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%% Purpose: Unix tar (tape archive) utility.
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
--export([init/3, create/2, create/3, extract/1, extract/2, table/1, table/2,
-	 open/2, close/1, add/3, add/4,
-	 t/1, tt/1, format_error/1]).
+-export([init/3,
+         create/2, create/3,
+         extract/1, extract/2,
+         table/1, table/2, t/1, tt/1,
+         open/2, close/1,
+         parse_octal/1,
```

Comment:
Ah, forgot to remove that during some testing, and it escaped my review, sorry about that!